src/js/health-records/containers/Main.jsx

```diff
@@ -179,6 +182,9 @@ export class Main extends React.Component {
     const missingDateError =
     [this.state.startDateError, this.state.endDateError]
       .indexOf('missing') > -1;
+    const startDateError =
+    [this.state.startDateError, this.state.endDateError]
```

Comment:
Used indexOf because I am sure that it is supported across browsers. Includes would fail in IE, but since I believe we have polyfills now that work I'll switch to it for readability